packages/core-mobile/app/screens/earn/Confirmation/Confirmation.tsx

```diff
@@ -415,6 +452,28 @@ export const Confirmation = (): JSX.Element | null => {
           </Tooltip>
           {renderNetworkFee()}
         </Row>
+        {xpProvider && xpProvider.isEtnaEnabled() && (
+          <>
+            <NetworkFeeSelector
+              chainId={pNetwork.chainId}
+              gasLimit={Number(defaultTxFee?.toSubUnit() ?? 1n)}
```

Comment:
if defaultTxFee is not available I don't think we should render this since using 1n for gasLimit will not work anyway